### PR TITLE
Add covid page related updates in emergency page

### PR DESCRIPTION
**Pull request by frozenhelium**
- Add Links section
- Add Fetaured Documents section
- Clean up UI for response documents

Depends on: https://github.com/IFRCGo/go-api/pull/1120

Related to #1950

### CODE REVIEW

File: src/root/views/emergency.js [hunk 1]
```diff
@@ -1222,6 +1249,45 @@ class Emergency extends React.Component {
...
+                                  rel="noopener noreferrer"
+                                >
+                                  {title}
+                                </a>
+                                <div className="emergency-link-description">
+                                  {description}
+                                </div>
+                              </div>
+                            );
+                          })}
```
Comment (AdityaKhatri) [inline src/root/views/emergency.js:1286]: Lets use List, this is complicated

File: src/root/views/emergency.js [hunk 2]
```diff
@@ -1286,14 +1352,69 @@ class Emergency extends React.Component {
...
+                                  >
+                                    {title}
+                                  </a>
+                                  <div className="featured-document-description">
+                                    {description}
+                                  </div>
+                                </div>
+                              </div>
+                            );
+                          })}
```
Comment (AdityaKhatri) [inline src/root/views/emergency.js:1397]: Lets use List, this is complicated

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Review by tnagorra — APPROVED** [review]

**Comment by batpad** [comment]
@frozenhelium this looks great to me --

Should we deploy this on a Surge URL and get feedback, and hold off on merging to `develop` just yet, in case for any reason we want to hold back on these changes for this release? (Though ideally we would just roll these in to this release, just might be nice to get an initial round of feedback on Surge "just in case" there's anything major?)

**Review by AdityaKhatri — APPROVED** [review]

**Comment by frozenhelium** [comment]
@batpad I think It'd be alright since this one doesn't have any major or breaking changes. The features are well separated so if we want we can easily revert back (with couple of comments)

**Comment by batpad** [comment]
> @batpad I think It'd be alright since this one doesn't have any major or breaking changes. The features are well separated so if we want we can easily revert back (with couple of comments) if necessary.

Great! Then let's just wait for when @vdeak has updating the staging backend and go ahead and merge this?

**Pull request merged**